metron-deployment/playbooks/metron_install.yml

```diff
@@ -15,13 +15,6 @@
 #  limitations under the License.
 #
 ---
-- hosts: metron
-  become: true
-  roles:
-    - { role: ambari_slave }
-    - { role: metron-builder, tags: ['build'] }
```

Comment:
This is the cause of the second build.  

The process was a bit more complex when Quick Dev was around because it would launch the Quick Dev image, then rebuild and try to push out new bits to Quick Dev.  Since we don't need that any longer, this can be simplified.

Reply:
Will --ansible-skip-tags="build"  still keep it from building at all?

Reply:
Yes